Tools/CISupport/ews-build/config.json

```
@@ -391,6 +395,12 @@
       "name": "Safe-Merge-Queue", "shortname": "safe-merge", "icon": "buildAndTest",
       "factory": "SafeMergeQueueFactory", "platform": "*",
       "workernames": ["webkit-cq-01", "webkit-cq-02", "webkit-cq-03", "webkit-cq-04", "webkit-cq-05"]
+    },
+    {
+      "name": "WPE-Skia-Build-EWS", "shortname": "wpe-skia", "icon": "buildOnly",
+      "factory": "WPEBuildFactory", "platform": "wpe",
+      "configuration": "release", "architectures": ["x86_64"],
```

We will want to pass `USE_SKIA=ON` to the build step, which could be done the [same way as for the post-commit queue, using additionalArguments](https://github.com/WebKit/WebKit/pull/24146/files):

```suggestion
      "configuration": "release", "architectures": ["x86_64"],
      "additionalArguments": ["--cmakeargs=-DUSE_SKIA=ON"],
```

---

It's not needed. Bots can set additional cmakeargs via BUILD_WEBKIT_ARGS env variable. I'm still fine with the change because since this a Skia bot setting the argument in the `config.json` file makes it more explicit.

---

I changed my mind, I think it's not a good idea to put cmakeargs arguments in the `config.json` bot.

Currently there's no other EWS bot that does that. As for `Tools/CISupport/build-webkit-org/config.json` there's currently only two bots that do that (both WPE bots).

The disadvantge of putting build arguments in the `config.json` file is that an build argument change (add, remove or modify build arguments) implies a commit in `config.json`. I believe that's the reason why we tend not to put build arguments in `config.json`.

---

I'd like to add a `--skia` argument to `build-webkit` so more on our side can start playing with it.

---

The equivalent change for https://github.com/WebKit/WebKit/pull/24146 didn't land in the end so I assume the same change for ews is not needed.